daemon/client.go

```diff
@@ -13,12 +13,13 @@ import (
 // ExecuteRequest request the daemon to execute a request
 func ExecuteRequest(commandLineOptions *cli.CommandLineOptions) (*request.ExecutedRequestResponse, error) {
 	daemonRequest := &Request{
-		Body:      commandLineOptions.Body,
-		Headers:   commandLineOptions.Headers,
-		Method:    commandLineOptions.Method,
-		Profiles:  commandLineOptions.Profiles,
-		URL:       commandLineOptions.URL,
-		Variables: commandLineOptions.Variables,
+		Body:        commandLineOptions.Body,
```

Comment:
It looks like `Request` and `CommandLineOptions` have the same structure? Can you use [`conversion`](https://stackoverflow.com/a/37246712/5225575) here?

Reply:
Same as above. I'll probably add something to do this in a more concise way in the future.